gui/default/index.html

```diff
@@ -860,6 +860,12 @@ <h4 class="panel-title">
                           </a>
                         </td>
                       </tr>
+                      <tr ng-if="deviceStatus(deviceCfg) == 'syncing'">
+                        <th><span class="fas fa-fw fa-hourglass-half"></span>&nbsp;<span translate>Time Remaining</span></th>
+                        <td class="text-right">
+                          <span>{{syncRemaining(deviceCfg.deviceID)}}</span>
```

Comment:
@spideysri2802 If `syncRemaining` is simplified to return the seconds remaining, you could use
```suggestion
                          <span>{{syncRemaining(deviceCfg.deviceID) | duration}}</span>
```
to translate the duration, to align with the other durations, as they are all translated, per https://github.com/syncthing/syncthing/pull/9552.

Reply:
I did try using "| duration" and "| binary" for the time and size in my js, to simplify the data and return, but it throws an error for some reason.. is this only usable on the html part of the code?